package.json

```diff
@@ -83,4 +83,4 @@
     "**/msgpackr": "1.8.3",
```

Comment:
The msgpackr resolution must be updated (or removed) as Theia also moved to a later version